### DISCUSSION TITLE
Aikido 3d for ubuntu

### DISCUSSION THREAD

**Discussion by damirturkovic** [community/community#74327] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Good afternoon. Can you help me with a possible possibility of shooting an aikido 3d program that I'd like to record on a button. What is the procedure and how to do it, if possible. I downloaded files from a github called aikido3d-master.

Damir Turković
Nice regards, everybody.

**Comment by damirturkovic** [comment]
I'd like a tutorial about shooting aikido 3d programs on the ubuntu.

**Comment by ettaboyle** [comment]
Hi @damirturkovic,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the project's repository. 

It's possible that another GitHub user might have run into this same issue and can help, but **the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas**. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
